### PR TITLE
Add Or.fromEither. Fixes #163

### LINKED ISSUE #163: Add Or.fromEither
Similar to `Or.fromTry`. `Either[A, B] => A Or B`.

### PR DESCRIPTION
**Pull request by bobbyrauchenberg**

### CODE REVIEW

File: data/src/main/scala/cats/data/Or.scala [hunk 1]
```diff
@@ -178,4 +178,10 @@ trait OrFunctions {
     case Failure(e) => left(e)
     case Success(v) => right(v)
   }
+
+  /**
+   * Convert an `Either[A, B]` to `A Or B`
+   *
```
Comment (julien-truffaut) [inline data/src/main/scala/cats/data/Or.scala]: small detail, can you remove this blank line?

File: data/src/main/scala/cats/data/Or.scala [hunk 2]
```diff
@@ -178,4 +178,10 @@ trait OrFunctions {
...
+   */
+  def fromEither[A, B](e: Either[A, B]): A Or B = e.fold(left(_), right(_))
```
Comment (julien-truffaut) [inline data/src/main/scala/cats/data/Or.scala]: does it compile if you remove `(_)` for left and right? e.g. `e.fold(left, right)`

**Comment by ceedubs** [comment]
:+1: thanks!

Line 184 could be removed to match the consistency of the other ScalaDoc comments in the file, but I know it doesn't actually matter, so I'm resisting my robotic consistency urge.

**Comment by bobbyrauchenberg** [comment]
sorry, i will remove it now... i know how painful these things can be!

@julien-truffaut - yes, it compiles. thanks for that.

**Comment by ceedubs** [comment]
:+1:

**Comment by julien-truffaut** [comment]
:+1:

**Pull request merged**